code/game/objects/structures/false_walls.dm

```diff
@@ -143,8 +143,11 @@
 /obj/structure/falsewall/get_dumping_location(obj/item/storage/source,mob/user)
 	return null
 
+/obj/structure/falsewall/examine(mob/user)
+	. += ..()
```

Comment:
Why is this += and not just =?

Reply:
copied code from deconstruction code e.g. https://github.com/yogstation13/Yogstation/blob/master/code/game/turfs/simulated/walls.dm#L48-L50